Javascript/exportimport.js

```diff
@@ -1,7 +1,14 @@
 function getRealTime(clock12h = false) {
     let now = new Date();
-    let date = `${now.getFullYear()}-${now.getMonth() + 1}-${now.getDate()}`
-    let time = now.toLocaleTimeString([], {hour12: clock12h});
+    let loc = "en"
```

Comment:
The "en" locale doesn't force us Americans to use the 12am -> 00 mapping. At the very least, update to `let loc = "EN-GB"` if you don't replace this with the suggestion I made.